<?xml version="1.0" encoding="UTF-8"?><record_update>
    <sys_documentation element="datasource_scope" label="Data Source Scope" language="en" table="x_ibmrt_guard_test_exception">
        <sys_documentation action="INSERT_OR_UPDATE">
            <element>datasource_scope</element>
            <help/>
            <hint/>
            <label>Data Source Scope</label>
            <language>en</language>
            <name>x_ibmrt_guard_test_exception</name>
            <plural>Data Source Scopes</plural>
            <sys_class_name>sys_documentation</sys_class_name>
            <sys_created_by>admin</sys_created_by>
            <sys_created_on>2022-01-21 21:17:16</sys_created_on>
            <sys_id>0e2958b51b374d50489ceb522a4bcbd6</sys_id>
            <sys_mod_count>2</sys_mod_count>
            <sys_name>Data Source Scope</sys_name>
            <sys_package display_value="IBM Guardium Vulnerability Assessment" source="x_ibmrt_guard">912d85fd87940110387c64280cbb35dc</sys_package>
            <sys_policy/>
            <sys_scope display_value="IBM Guardium Vulnerability Assessment">912d85fd87940110387c64280cbb35dc</sys_scope>
            <sys_update_name>sys_documentation_x_ibmrt_guard_test_exception_datasource_scope_en</sys_update_name>
            <sys_updated_by>admin</sys_updated_by>
            <sys_updated_on>2022-04-29 12:24:03</sys_updated_on>
            <url/>
            <url_target/>
        </sys_documentation>
    </sys_documentation>
</record_update>
